bcda/cli.go

```diff
@@ -200,13 +231,23 @@ func importCCLF9(fileMetadata cclfFileMetadata) error {
 	for sc.Scan() {
 		b := sc.Bytes()
 		if len(bytes.TrimSpace(b)) > 0 {
-			fmt.Printf("\nXREF: %s\n", b[0:1])
-			fmt.Printf("Current identifier: %s\n", b[currIDStart:currIDEnd])
-			fmt.Printf("Previous identifier: %s\n", b[prevIDStart:prevIDEnd])
-			fmt.Printf("Previous identifier effective date: %s\n", b[prevIDEffDateStart:prevIDEffDateEnd])
-			fmt.Printf("Previous identifier obsolete date: %s\n", b[prevIDObsDateStart:prevIDObsDateEnd])
+
+			cclf9 := models.CCLF9{
+				XrefIndicator: string(b[0:1]),
+				CurrentNum:    string(b[currIDStart:currIDEnd]),
+				PrevNum:       string(b[prevIDStart:prevIDEnd]),
+				PrevsEfctDt:   string(b[prevIDEffDateStart:prevIDEffDateEnd]),
+				PrevsObsltDt:  string(b[prevIDObsDateStart:prevIDObsDateEnd]),
+			}
+			err = db.Create(&cclf9).Error
+			if err != nil {
+				return errors.Wrap(err, "could not create CCLF9 record")
```

Comment:
Same here with naming, it sounds like the file couldn't be stored rather than a row from it.